### ISSUE TITLE
Reverse for 'nesting_server_data' not found

### ISSUE THREAD

**Issue by bparker06** [theatlantic/django-nested-admin#170]
I just installed django-nested-admin 3.4.2 via pip with django 2.2 and followed the instructions in the README (not using grappelli).

Trying to navigate to an admin changelist form gives the error `Reverse for 'nesting_server_data' not found. 'nesting_server_data' is not a valid view function or pattern name.`.

Any idea what could be wrong?

**Comment by fdintino** [comment]
Hi. I do know the issue. The problem is that I need to make a release to PyPI; the README refers to the current code on master, but I haven't yet pushed out a release. I'll find some time to do it today.

**Comment by fdintino** [comment]
This is fixed in v3.3.0. Please reopen if you still are experiencing this issue. Thanks!

**Comment by tvanesse** [comment]
I'm still having this issue with the following config:
```
Django==4.1.2
django-nested-admin==4.0.2
django-grappelli==3.0.4
```
